docs/source/_gallery_original/index.rst

```diff
@@ -35,6 +35,7 @@ Contents
    :caption: LLM Models
 
    Mixtral (Mistral AI) <llms/mixtral>
+   MiniCPM (openbmb) <llms/minicpm>
```

Comment:
By adding this here, we need to soft link the readme file at `llms/minicpm/README.md` to `docs/source/_gallery_original/llms/`.

Reply:
Already modified